Language/Functions/Interrupts/interrupts.adoc

```diff
@@ -17,10 +17,9 @@ subCategories: [ "Interrupts" ]
 
 [float]
 === Description
-Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code.
+Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code. They can also cause races, code that behaves differently, and incorrectly, in response to tiny changes in timing. See the link:../External%20Interrupts/attachInterrupt[attachInterrupt()] page for further information.
```

Comment:
```suggestion
Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code. They can also cause races, code that behaves differently, and incorrectly, in response to tiny changes in timing. See the link:../../external-interrupts/attachinterrupt[attachInterrupt()] page for further information.
```

Correct link.

Each file is converted to a folder on the website, so this requires an extra tree level in links when compared to the paths here in the repository.

All characters in the paths are lowercased on the website.

The website uses a hypen as a space replacement character (e.g., `external-interrupts`)